internal/powerbi/resource_pbix.go

```diff
@@ -323,6 +339,13 @@ func readImport(d *schema.ResourceData, meta interface{}, timeoutForSuccessfulIm
 		if len(im.Reports) >= 1 {
 			d.SetPartial("report_id")
```

Comment:
I think the `report_id` and `dataset_id` can now be moved out of the if statement here, to always correctly reflect their values in state. Only the `report_original_dataset_id` should only be set if this is indeed a new resource.